util/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -211,9 +213,20 @@ protected Helper delegate() {
     public Subchannel createSubchannel(CreateSubchannelArgs args) {
       // Subchannels are wrapped so that we can monitor call results and to trigger failures when
       // we decide to eject the subchannel.
-      OutlierDetectionSubchannel subchannel = new OutlierDetectionSubchannel(
-          delegate.createSubchannel(args));
-
+      HealthUtil.HealthCheckingListener hcListener = args.getOption(HEALTH_LISTENER_ARG_KEY);
+      HealthUtil.ChainedHealthListener outlierListener = null;
+      if (hcListener != null ) {
+        HealthUtil.ChainedHealthListener chainedHealthListener =
+            new HealthUtil.ChainedHealthListener(hcListener);
+        args = args.toBuilder()
+            .addOption(HEALTH_LISTENER_ARG_KEY, chainedHealthListener).build();
+        if (hcListener.getGeneration() > 0) {
+          outlierListener = chainedHealthListener;
```

Comment:
If you are going to drop it because of no upstream you need to at least log a warning.